root/static/scripts/release-editor/init.js

```diff
@@ -260,6 +267,71 @@ releaseEditor.init = function (options) {
     getRecordings();
   });
 
+  /**
+   * Check for similarly-named existing release groups when the release
+   * title or artist credits change.
+   */
+  let duplicateRGsRequest = null;
+  let duplicateRGsQuery = null;
+  debounceComputed(utils.withRelease((release) => {
+    const query = utils.constructLuceneFieldConjunction({
+      arid: release.artistCredit().names
+        .map((a) => a.artist?.gid)
+        .filter(Boolean),
+      releasegroup: [`"${utils.escapeLuceneValue(release.name() ?? '')}"`],
```

Comment:
It seems like it'd be better to not quote the search term, since forcing a single phrase will fail to find very similar names in many cases; for example, currently, entering "Il Buono, Il Brutto E Il Cattivo" as the release title and "Ennio Morricone" as the artist will not find any existing RGs, while using "Il buono, il brutto, il cattivo" as the title will.

Reply:
Thanks for the explanation -- I've removed the quotes. I wasn't sure how the results are ranked, but it hopefully doesn't matter too much since the search is limited to the credited artists.